server.py

```diff
@@ -87,6 +88,34 @@ def to_dict(self):
         return {c.name: getattr(self, c.name) for c in self.__table__.columns}
 
 
+class FeedbackResponse(db.Model):
+    __tablename__ = "feedback_responses"
+    id = db.Column("id",
+                   Integer,
+                   primary_key=True)
+
+    ts = db.Column("ts",
+                   db.TIMESTAMP(timezone=True),
+                   server_default=db.func.now(),
+                   onupdate=db.func.now())
+
+    invalid = db.Column("invalid",
+                        db.Boolean,
+                        default=False)
+
+    data = db.Column("data", JSONB)
+
+    survey = db.Column("survey", String)
+
+    period = db.Column("period", String)
+
+    def __init__(self, invalid, data, survey, period):
+        self.invalid = invalid
+        self.data = data
+        self.survey = survey
+        self.period = period
+
```

Comment:
We'll need to make sure this table exists in the preprod and prod databases before we deploy